docs/install/lin/activating-a-validator.md

```diff
@@ -3,24 +3,29 @@ id: activating-a-validator
 title: Activating a Validator
 sidebar_label: Activate a validator
 ---
-This section outlines the step-by-step process found on prylabs.net to submit a deposit and initialise a validator for participation in the testnet.
+This section outlines the step-by-step process for Linux found on prylabs.net to submit a deposit and initialise a validator for participation in the testnet.
```

Comment:
Linux is referred to as GNU/Linux throughout the manual, though not overly important either

Reply:
fixed & noted